user/storage.md

```diff
@@ -13,29 +13,45 @@ The following material was adapted from the
 [Pangeo Cloud User Guide](https://pangeo.io/cloud.html)
 :::
 
-## Your Home Directory
+## The JupyterHub Filesystem
 
 Your notebook server is a linux "virtual machine" with its own filesystem.
-Your are not on a shared server; you are on your own private server.
-Your username is ``jovyan``, and your home directory is ``/home/jovyan``.
-This is the same for all users.
-
-Your home directory is intended only for notebooks, analysis scripts, and small datasets (< 1 GB).
-It is not an appropriate place to store large datasets.
-No one else can see or access the files your home directory.
+You are not on a shared server; you are on your own private server.
 
 The easiest way to move files in and out of your home directory is via the JupyterLab web interface.
 Drag a file into the file browser to upload, and right-click to download back out.
 You can also open a terminal via the JupyterLab launcher and use this to ssh / scp / ftp to remote systems.
 However, you can’t ssh in!
 
-## The `shared` Directory
+### Your Home Directory
+
+Your username is ``jovyan``, and your home directory is ``/home/jovyan``.
+This is the same for all users, but no one else can see or access the files in *your* home directory.
+
+``/home/jovyan`` is a persistant network-attached drive. Any files you put there will be there when you
+log out and log back into the JupyterHub. 
+
+The ``/home/jovyan`` space is typically limited to 10 GB. Consequently, your home directory is intended 
+only for notebooks, analysis scripts, and small datasets (< 1 GB). It is not an appropriate place to store 
+large datasets.
```

Comment:
Is this wording an OK compromise? seems there is a desire to not lie about enforced limits but still provide guidelines to users on ballpark maximum file storage... as a concrete example, we commonly have people wanting to store 100s of 500MB satellite images somewhere.

Reply:
I like it, +1 from me